UnitsNet/UnitDefinitions/HeatFlux.json

```diff
@@ -0,0 +1,117 @@
+﻿{
+  "Name": "HeatFlux",
+  "BaseUnit": "WattPerSquareMeter",
+  "XmlDoc": "Heat flux is the flow of energy per unit of area per unit of time",
+  "Units": [
+    {
+      "SingularName": "WattPerSquareMeter",
+      "PluralName": "WattsPerSquareMeter",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Prefixes": [ "Nano", "Micro", "Milli", "Centi", "Deci", "Kilo" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "W/m²" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "WattPerSquareInch",
+      "PluralName": "WattPerSquareInch",
+      "FromUnitToBaseFunc": "x*1.5500031e3",
+      "FromBaseToUnitFunc": "x/1.5500031e3",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "W/in²" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "WattPerSquareFoot",
+      "PluralName": "WattPerSquareFoot",
```

Comment:
Plural: Watts

Reply:
Use SingularName for BaseUnit. ;)

Reply:
BaseUnit needs to be singular, plural is only used to generate proper names of methods and properties, such as `double val = myLength.Meters` and `double val = myHeatFlux.WattsPerSquareFoot`.

Reply:
Excellent, thanks for the info. Good to get acquainted as I plan to contribute in the future :)